### PR TITLE
Updated DS Mentor/Mentee schema to match backend

### PR DESCRIPTION
**Pull request by yefim-g**
## Description

DS needs to match BE Mentor and Mentee creation to match their schema and store it.

## Type of change

- [X] New feature (updated)

## Checklist:

- [X] My code follows PEP8 style guide
- [X] I have removed unnecessary print statements from my code
- [X] I have made corresponding changes to the documentation if necessary
- [X] My changes generate no errors
- [x] No commented-out code
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

## Pair-Programming:
Who you paired with: Matt Grohnke
What pairing method(s) you used: Driver-navigator
How did it go? What did you learn?: The process went well, it sped up the time that it would have taken to make the updates.

Loom video: https://www.loom.com/share/0878bfa264174888b6b3aad964e5e909

### CODE REVIEW

**Review by NPAPENBURG — APPROVED** [review]
Good changes. Allows for our data to align with what is being sent from the backend.

File: data_generators/user_generators.py [hunk 1]
```diff
@@ -8,60 +8,65 @@
...
         self.industry_knowledge = percent_true(15)
         if self.job_help:
             self.pair_programming = False
         else:
             self.pair_programming = percent_true(60)
-        self.other_info = "Notes"
+        self.your_hope = 'String'
         self.need = choice(resource_items)
-        self.parole_restriction = choice(parole_restriction)
+        self.parole_restriction = ''
```
Comment (BrokenShell) [inline data_generators/user_generators.py:65]: line 65: I found out this should be a random bool... we have a function for that:
```
self.parole_restriction = percent_true(50)
```

**Review by BrokenShell — CHANGES_REQUESTED** [review]
see line 65 in the Mentee generator

**Review by NPAPENBURG — APPROVED** [review]
Everything was cleaned up perfect.

**Review by MichaelJP-DS — APPROVED** [review]
Nice fix - Looks good to go!

**Review by jonathan-mcgraw — APPROVED** [review]
Looks great, sir. It will be nice to have databases across all 3 repos with matching schemas.

**Review by jinjahninjah — CHANGES_REQUESTED** [review]
Great job Yefim and Matt! The only changes I would suggest is keeping the doc string quotes to double quotes in the Mentor, Mentee, and Meeting classes, but other than that looks awesome!

**Comment by Zack-Quintana** [comment]
Yeah, looks good. No fixes need to be implemented that I can think of. nice job

**Review by jinjahninjah — APPROVED** [review]

**Review by BrokenShell — APPROVED** [review]

**Pull request merged**